images/windowsserver2022/windows-server-2022.json

```diff
@@ -0,0 +1,74 @@
+{
+    "builders": [
+        {
+            "type": "hyperv-iso",
+            "iso_url": "https://software-static.download.prss.microsoft.com/sg/download/888969d5-f34g-4e03-ac9d-1f9786c66749/SERVER_EVAL_x64FRE_en-us.iso",
+            "iso_checksum": "3e4fa6d8507b554856fc9ca6079cc402df11a8b79344871669f0251535255325",
+            "floppy_files": ["autounattend.xml"],
+            "vm_name": "packer-build-Win2022",
+            "boot_wait": "5s",
+            "disk_size": "10240",
+            "headless": false,
+            "winrm_password" : "packer",
+            "winrm_username" : "Administrator",
+            "communicator": "winrm",
+            "winrm_use_ssl": true,
+            "winrm_insecure": true,
+            "winrm_timeout": "4h",
+            "shutdown_timeout": "30m",
+            "shutdown_command": "shutdown /s /t 5 /f /d p:4:1 /c \"Packer Shutdown\"",
+            "skip_compaction": true,
+            "switch_name": "VmNAT",
+            "generation": 1,
+            "use_fixed_vhd_format": true
+        }
+    ],
+    "provisioners": [
+        {
+            "type": "powershell",
+            "scripts": ["scripts/setup-1.ps1"]
+        },
+        {
+            "type": "windows-restart",
+            "restart_timeout": "30m"
+        },
+        {
+            "type": "powershell",
+            "scripts": ["scripts/setup-2.ps1"],
+	        "pause_before": "1m"
+        },
+        {
+            "type": "windows-restart",
+            "restart_timeout": "30m"
+        },
+        {
+            "type": "powershell",
+            "scripts": ["scripts/setup-3.ps1"]
+        },
+        {
+            "type": "windows-restart",
+            "restart_timeout": "30m"
+        },
+        {
+            "destination": "\\Windows",
+            "source": "C:\\Users\\azureuser\\Downloads\\",
+            "type": "file"
+        },
+        {
+            "inline": [
+                "Start-Process \\Windows\\WindowsAzureVmAgent.2.7.41491.1018_2107161018.fre.msi"
+            ],
+            "type": "powershell"
```

Comment:
nit: can we keep the same property ordering of "type" first, then other properties following for all provisioners to make it easy to read "what is this?", followed by "and here's how that is configured"?